sunpy/map/mapbase.py

```diff
@@ -166,7 +159,21 @@ class GenericMap(NDData):
     """
 
     def __init__(self, data, header, plot_settings=None, **kwargs):
-
+        """
+        Parameters
+        ----------
+        data : `~numpy.ndarray`, list
+            A 2d list or ndarray containing the map data.
+        header : dict
```

Comment:
can you add backticks around dict.
```
`dict`
```

Reply:
The objects automatically get linked here in numpydoc strings without the backticks if that's what you were thinking?